src/build.ts

```diff
@@ -125,6 +129,31 @@ export async function build(
   if (pageCount) effects.logger.log(`${faint("built")} ${pageCount} ${faint(`page${pageCount === 1 ? "" : "s"} in`)} ${root}`); // prettier-ignore
   if (assetCount) effects.logger.log(`${faint("built")} ${assetCount} ${faint(`asset${assetCount === 1 ? "" : "s"} in`)} ${root}`); // prettier-ignore
 
+  // Check links.
+  const REF = "proto://host/_observablehq";
```

Comment:
This is needed to detect links that try to go higher up than the root. I'm using `_observablehq/` as the pathname since it's a reserved word — and we should never encounter a link to root/_observablehq/page.md.